<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordRequest</class>
 <widget class="QWidget" name="PasswordRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>80</y>
     <width>491</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="MyLabelAnimated" name="pb_ok">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Bold</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     border: 2px solid #E3E3E3;
     border-radius: 5px;
     color: #E3E3E3
 }</string>
    </property>
    <property name="text">
     <string>ACEPTAR</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Bold</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: rgb(35, 33, 30);
     border-radius: 10px;
     color: #E3E3E3
 }</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_nombre_usuario">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>23</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #E3E3E3</string>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MyLineEdit" name="le_nombre_usuario">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>50</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>23</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
    </property>
    <property name="text">
     <string>NEURONIC</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MyLineEdit" name="le_clave">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>136</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>23</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     border-radius: 3px;
     background-color: #E3E3E3;
     color: #444444
 }</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="l_clave">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>110</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #E3E3E3</string>
    </property>
    <property name="text">
     <string>Contrase√±a</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MyLabelAnimated" name="pb_show_hide_password">
    <property name="geometry">
     <rect>
      <x>417</x>
      <y>137</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DIN Pro Cond Bold</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/show.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>le_clave</zorder>
   <zorder>pb_ok</zorder>
   <zorder>pb_show_hide_password</zorder>
   <zorder>l_clave</zorder>
   <zorder>le_nombre_usuario</zorder>
   <zorder>l_nombre_usuario</zorder>
  </widget>
  <widget class="QLabel_Button" name="l_background_blur">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="assets.qrc">:/icons/menu_on.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>l_background_blur</zorder>
  <zorder>widget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabelAnimated</class>
   <extends>QLabel</extends>
   <header location="global">mylabelanimated.h</header>
  </customwidget>
  <customwidget>
   <class>QLabel_Button</class>
   <extends>QLabel</extends>
   <header location="global">qlabel_button.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
